### PR TITLE
add 9 icons

### PR DESCRIPTION
**Pull request by piecesuger**
# Description

Add icons for Live2DViewerEX, CaptiveMgr, Ruler, Spectroid, Starth Bing Wallpaper, Only One, Icon Pusher, idolmaster million live theater days, QooApp, 
### Icons added
* Live2DViewerEX (`com.pavostudio.live2dviewerex`)
* CaptiveMgr (`tech.evlsoc.captivemgr`)
* Ruler (`fr.ecp.ruler.app`)
* Spectroid (`org.intoorbit.spectrum`)
* Starth Bing Wallpaper (`me.liaoheng.wallpaper`)
* Only One (`com.onlyone.onlyonestarter`)
* Icon Pusher(`dev.southpaw.iconpusher`)
* idolmaster million live theater days (`{com.bandainamcoent.imas_millionlive_theaterdays`)
* QooApp (`com.qooapp.qoohelper`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Review by x9136 — CHANGES_REQUESTED** [review]
Thank you for contributing!

It's good that you did everything with a stroke.

All icons have wrong dimensions. There are other flaws as well: incorrect stroke widths (e.g. 9px in the `only_one.svg`), wrong rounding of right corners (`icon_pusher.svg`, `live2dviewerex.svg`, `qooapp.svg` and `ruler.svg`). If you plan on fixing this, it's worth spending 10 minutes [on the guides](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#artboard--grid). It will help and speed up the acceptance of the pull request. 

I also want to draw your attention to visual flaws: a too bold dot on the `starth.svg` icon or a clumped "e" on the `only_one.svg` icon. The `spectroid.svg` should have a slightly thicker stroke (14px) because the icon takes up very little space. I suggest to leave it for later, it is more important to make the icons fit the guides first.

File: app/assets/appfilter.xml
[the remark this answers is not in the document]
Comment (piecesuger) [reply app/assets/appfilter.xml:1]: I have made some modifications. By the way what do you mean wrong dimensions here? Size of whole icon?

[the remark this answers is not in the document]
Comment (x9136) [reply app/assets/appfilter.xml:1]: > By the way what do you mean wrong dimensions here? Size of whole icon?

Yes, the size of the icon.
Screenshot 2023-10-27 at 09 15 12

[the remark this answers is not in the document]
Comment (x9136) [reply app/assets/appfilter.xml:1]: After the fixes, the sizing problem remained. Also note that you have formatted appfilter.xml differently than on the screenshot.

[the remark this answers is not in the document]
Comment (piecesuger) [reply app/assets/appfilter.xml:1]: OK. Already resized and formatted.

File: svgs/only_one.svg
Comment (x9136) [inline svgs/only_one.svg:1]: Screenshot 2023-10-27 at 16 08 54
The size is a little smaller than it needs to be.

If you want to work on quality, I suggest you make the "e" differently and raise the word a bit higher so it doesn't press down.

File: svgs/qooapp.svg
Comment (x9136) [inline svgs/qooapp.svg:1]: Screenshot 2023-10-27 at 16 10 37
The size is a little smaller than it needs to be. It is also worth moving the icon to the left to balance the empty space (I did it on the screenshot).

File: svgs/ruler.svg
Comment (x9136) [inline svgs/ruler.svg:1]: The size is a bit wrong, but it'll do for the first time.

File: svgs/spectroid.svg
Comment (x9136) [inline svgs/spectroid.svg:1]: Screenshot 2023-10-27 at 16 16 11
The size is a little smaller than it needs to be, but it'll do for the first time. 

If you want to work on the quality, it is worth raising the icon a bit higher to compensate for the empty space.

File: svgs/sse.svg
Comment (x9136) [inline svgs/sse.svg:1]: Screenshot 2023-10-27 at 16 17 58
The size is a little smaller than it needs to be, but it'll do for the first time.

If you want to work on quality, it is worth raise the icon a little higher and avoid small spacing between strokes.

File: svgs/starth.svg
Comment (x9136) [inline svgs/starth.svg:1]: Screenshot 2023-10-27 at 16 20 51
The size is a little smaller than it needs to be, but it'll do for the first time.

If you want to work on quality, it is worth raise the icon a little higher.

File: svgs/captivemgr.svg
[the remark this answers is not in the document]
Comment (x9136) [reply svgs/captivemgr.svg:1]: Screenshot 2023-10-27 at 16 24 50
If you need to reply in a specific thread, click here first.

**Review by x9136 — CHANGES_REQUESTED** [review]
All icons need a little tweaking, but some of them can be accepted as is. I have left comments on each icon separately.

**Comment by piecesuger** [comment]
> All icons need a little tweaking, but some of them can be accepted as is. I have left comments on each icon separately.

Done.

**Review by x9136 — APPROVED** [review]

**Pull request merged**